doc/lua_api.txt

```diff
@@ -2477,7 +2477,27 @@ For the following functions, `v`, `v1`, `v2` are vectors,
     * Returns a boolean, `true` if the vectors are identical.
 * `vector.sort(v1, v2)`:
     * Returns in order minp, maxp vectors of the cuboid defined by `v1`, `v2`.
-
+* `vector.random_2d()`
+    * Returns a new normalized 2d vector with random values
+* `vector.random_3d()`
+    * Returns a new normalized 3d vector with random values
+* `vector.set_length(v, l)`
+    * Returns a vector with a specific length (magnitude)
+* `vector.limit(v, l)`
+    * Limits the length of a vector
+* `vector.dot(v1, v2)`
+    * Returns the dot product of two vectors
+* `vector.cross(v1, v2)`
+    * Returns the cross product of two vectors
+* `vector.scalar_triple(v1, v2, v3)`
+    * Returns the scalar triple product of three vectors
+* `vector.vector_triple(v1, v2, v3)`
+    * Returns the vector triple product of three vectors
```

Comment:
I've only found this [here](https://en.wikipedia.org/wiki/Triple_product#Tensor_calculus) on wikipedia but I don't know what it can be used for yet:
![formular](https://wikimedia.org/api/rest_v1/media/math/render/svg/cc35962348c89fb5242a06051f590758972f0a06)

Reply:
You given a link to the “tensor calculus” section =)
The reality is that it returns v1×(v2×v3), that is, a vector in the (v2, v3) plane orthogonal to v1.

Reply:
Ah, thanks for clarifying my stupidity! : )